crates/biome_js_analyze/src/lint/nursery/no_suspicious_semicolon_in_jsx.rs

```diff
@@ -42,14 +42,14 @@ declare_rule! {
     /// }
     /// ```
     ///
-    pub NoSemicolonInJsx {
+    pub NoSuspiciousSemicolonInJsx {
         version: "1.6.0",
```

Comment:
Sorry for not being able to point this out early. I'm not sure if we should update this? @ematipico

Reply:
You're right. This new be `"next"` instead of `"1.6.0"`. @fujiyamaorange can you apply the change and run `just gen-lint` again?